main.c

```diff
@@ -945,6 +946,10 @@ main
   notify_observer_add(NeoMutt->notify, NT_CONFIG, main_log_observer, NULL);
   notify_observer_add(NeoMutt->notify, NT_CONFIG, main_config_observer, NULL);
 
+  const char *c_devel_new_mail_command = cs_subset_string(NeoMutt->sub, "devel_new_mail_command");
+  notify_observer_add(NeoMutt->notify, NT_ALL, new_mail_observer,
+                      (void *) c_devel_new_mail_command);
+
```

Comment:
This will work if `$devel_new_mail_command` is defined in the config file, but not if it's turned on later.
Worse still, if the user changes the variable, then the observer will be passed a dangling pointer.

If you just pass `NULL` to `notify_observer_add()`, then we can check the value of `$devel_new_mail_command` in the observer.

Reply:
Right, didin't think of changing the value, very good point.